gatorgrade/main.py

```diff
@@ -0,0 +1,32 @@
+"""Use Typer to run gatorgrade to run the checks and generate the yml file."""
+import typer
+from pathlib import Path
+
+app = typer.Typer(add_completion=False)
+FILE_DIR = "config"
+FILE = "gatorgrade.yml"
+
+
+@app.callback(invoke_without_command=True)
+def gatorgrade(
+    filepath: Path = typer.Option(
+        f"{FILE_DIR}/{FILE}", "--path", "-p", help="Filepath to the yml file."
```

Comment:
Instead of referring to the configuration filepath here as `path`, you should chose a more descriptive argument name, such as `--config` or `-c` (or both).